Data/DataPreferences.swift

```diff
@@ -22,4 +22,21 @@ extension Preferences {
         /// See Sync.syncSeed for more details.
         static let seedName = Option<Bool>(key: "sync.is-sync-seed-set", default: false)
     }
+    
+    final class Database {
+        
+        public final class DocumentToSupportDirectoryMigration {
+            /// This is the filepath used for the coredata database
```

Comment:
Is this comment relevant? the preference is a bool, not a filepath string

Reply:
Good catch, fixed. I originally had this just pointing to filepath, but switched to bool. Thanks 😄